### PR TITLE
update mojo runtime in support-1.6 branch

### PR DESCRIPTION
**Pull request by thirdwing**
I am not sure this is the correct way to update mojo runtime and make a new release for DAI.

Please let me know if there are other things I need to do.

### CODE REVIEW

**Comment by osery** [comment]
@thirdwing ideally, we make the change separate from the release for nicer git history and archeology.
That would be this PR without the version change.

Next, we need to bump the version in the `support-1.6` branch to `SNAPSHOT-0.0.10` and create a branch `release-0.0.9` with a commit setting version to `0.0.9` (no snapshot here).

Normally, I tend to do this by two commits in the support-1.6 branch, one setting version to `0.0.9` and another setting it to `SNAPSHOT-0.0.10`. And then creating the `release-0.0.9` branch pointing to the first of the two commits. That will trigger the release process. Last, I also do a GitHub release (manually in the GitHub UI).

**Comment by thirdwing** [comment]
OK. I will do that.

File: gradle.properties [hunk 1]
```diff
@@ -4,7 +4,7 @@ version = 0.0.8-SNAPSHOT
 # issue resolution.
 
 # Internal dependencies:
-mojoRuntimeVersion = 0.13.18
+mojoRuntimeVersion = 0.13.20
```
Comment (thirdwing) [inline gradle.properties]: The only change in this PR is the mojo runtime version update.

Comment (osery) [reply gradle.properties]: :+1:.

**Comment by thirdwing** [comment]
The `release-0.0.9` branch has been created: https://github.com/h2oai/dai-deployment-templates/commit/6f936e2612b1c78dc8f26bdfc9ceefe73c9a80bb

**Comment by osery** [comment]
@thirdwing great, thanks.

**Comment by osery** [comment]
I have created the same release on GitHub just to be consistent:
https://github.com/h2oai/dai-deployment-templates/releases/tag/v0.0.9

**Pull request merged**